Lombiq.Tests.UI/SecurityScanning/SecurityScanConfiguration.cs

```diff
@@ -90,7 +96,7 @@ public SecurityScanConfiguration SignIn(string userName = DefaultUser.UserName)
     /// </param>
     public SecurityScanConfiguration ExcludeUrlWithRegex(string excludedUrlRegex)
     {
-        ExcludedUrlRegexPatterns.Add(excludedUrlRegex);
+        _excludedUrlRegexPatterns.Add(excludedUrlRegex);
```

Comment:
We pass this configuration into a ZAP, which is a Java application. The regex between .NET and Java isn't 100% compatible. ZAP will return an error if the regex is invalid. Such error will fail the test and enter into the test log. So this check is redundant at best. Could be even harmful in case the expression is valid in Java's regex flavor but not in .NET's regex flavor.